docs/installation.rst

```diff
@@ -117,6 +117,84 @@ the old version using the same steps as shown above.
 .. _pyinstaller: http://www.pyinstaller.org
 .. _releases: https://github.com/borgbackup/borg/releases
 
+.. _platforms:
+
+Features & platforms
+--------------------
+
+Besides regular file and directory structures, |project_name| can preserve
+
+    * Hardlinks (considering all files in the same archive)
+    * Symlinks (stored as symlink, the symlink is not followed)
+    * Special files:
+
+        * Character and block device files (restored via mknod)
+        * FIFOs ("named pipes")
+        * Special file *contents* can be backed up in ``--read-special`` mode.
+          By default the metadata to create them with mknod(2), mkfifo(2) etc. is stored.
+    * Timestamps in nanosecond precision: mtime, atime, ctime
+    * Permissions:
+
+        * IDs of owning user and owning group
+        * Names of owning user and owning group (if the IDs can be resolved)
+        * Unix Mode/Permissions (u/g/o permissions, suid, sgid, sticky)
+
+On some platforms additional features are supported:
+
+.. Yes/No's are grouped by reason/mechanism/reference.
+
++------------------+------------+----------+-----------+------------+
+| Platform         | Tested     | ACLs     | xattr     | Flags      |
+|                  |            | [#acls]_ | [#xattr]_ | [#flags]_  |
++==================+============+==========+===========+============+
+| Linux x86        | Travis-CI_ | Yes      | Yes       | Yes [1]_   |
++------------------+------------+          |           |            |
+| Linux PowerPC    | Golem_     |          |           |            |
++------------------+            |          |           |            |
+| Linux ARM        |            |          |           |            |
++------------------+------------+----------+-----------+------------+
+| Mac OS X         | Travis-CI_ | Yes      | Yes       | Yes (all)  |
++------------------+------------+----------+-----------+            |
+| FreeBSD          | Golem_     | Yes      | Yes       |            |
++------------------+            +----------+-----------+            |
+| OpenBSD          |            | n/a      | n/a       |            |
++------------------+            +----------+-----------+            |
+| NetBSD           |            | n/a      | No [2]_   |            |
++------------------+            +----------+-----------+------------+
+| Solaris 11       |            | No [3]_              | n/a        |
++------------------+            |                      |            |
+| OpenIndiana      |            |                      |            |
++------------------+------------+----------+-----------+------------+
+| Windows (native) | AppVeyor_  | Yes      | ?         | ?          |
++------------------+------------+----------+-----------+------------+
+| Windows (cygwin) | Manually   | No [4]_  | No        | No         |
++------------------+------------+----------+-----------+------------+
```

Comment:
Hmm, maybe the test sites could go into a separate section and just keep the supported features here?